integreat_cms/cms/migrations/0079_alter_poi_category.py

```diff
@@ -0,0 +1,61 @@
+# Generated by Django 3.2.20 on 2023-07-25 15:46
+
+from django.db import migrations, models
+
+from ..constants import poicategory
+
+
+# pylint: disable=unused-argument
+def check_and_update_poicategory(apps, schema_editor):
+    """
+    This function checks whether the default POI category "Other" exists and creates it if not.
+    It checks furthermore whether each location has a category and assigns the default category if not.
+    """
+    Language = apps.get_model("cms", "Language")
+    if not (default_language := Language.objects.first()):
+        # If no languages exists, we can assume that also no POIs exist
+        return
+
+    POICategory = apps.get_model("cms", "POICategory")
+    POI = apps.get_model("cms", "POI")
+
+    if not (
+        default_category := POICategory.objects.filter(icon=poicategory.OTHER).first()
+    ):
+        default_category, created = POICategory.objects.get_or_create(
+            icon=poicategory.OTHER,
+            defaults={"color": poicategory.DARK_BLUE},
+        )
+        if created:
+            POICategoryTranslation = apps.get_model("cms", "POICategoryTranslation")
+            POICategoryTranslation.objects.create(
+                category=default_category,
+                language=default_language,
+                name=poicategory.OTHER,
+            )
```

Comment:
```suggestion
        default_category = POICategory.objects.create(
            icon=poicategory.OTHER,
            color=poicategory.DARK_BLUE,
        )
        POICategoryTranslation = apps.get_model("cms", "POICategoryTranslation")
        POICategoryTranslation.objects.create(
            category=default_category,
            language=default_language,
            name=poicategory.OTHER,
        )
```
Ah, I think if no "other" category exists, it makes no sense to use `get_or_create()`, since we will create one in any case...
Sorry, in my last suggestion, I forgot that we agreed on the fact that there might be multiple other categories :innocent: